pyscada/utils/scheduler.py

```diff
@@ -704,7 +712,7 @@ def restart(self):
 
 
 class SingleDeviceDAQProcess(Process):
-    def __init__(self, dt=5, **kwargs):
+    def __init__(self, dt=1, **kwargs):
```

Comment:
the dt value can be overwritten with the dt_set value under the process config under DEVICE/__init__.py
```
parent_process_list = [{'pk':PROTOCOL_ID,
                        'label': 'pyscada.visa',
                        'process_class': 'pyscada.visa.worker.Process',
                        'process_class_kwargs': '{"dt_set":30}',
                        'enabled': True}]
```

Reply:
and via the Device model `polling_interval` field